lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -554,7 +578,7 @@ def self.example_code
             api_token: "...",
             owner_name: "appcenter_owner",
             app_name: "testing_app",
-            aab: "./app.aab",
+            file: "./app.aab",
             destinations: "Alpha",
             destination_type: "store",
             build_number: "3",
```

Comment:
Looking at this now I can see that the examples don't make much sense. aab doesn't need build_number, nor does apk.